src/components/Inputs/CheckBoxInput/CheckBoxInput.scss

```diff
@@ -0,0 +1,58 @@
+.checkbox-input {
+    &__container {
+        display: block;
+        position: relative;
+        padding-left: 23px;
+        cursor: pointer;
+        font-size: 20px;
+        -webkit-user-select: none;
+        -moz-user-select: none;
+        -ms-user-select: none;
+        user-select: none;
+        color: #FFFFFF;
+
+        &:hover input~.checkbox-input__checkmark {
+            background-color: #E44C5A;
+        }
+
+        input:checked~.checkbox-input__checkmark:after {
+            display: block;
+        }
```

Comment:
`input {&:checked~ { .... }}`

Reply:
Thank you for your comment, converted to scss